app/build.gradle

```diff
@@ -82,6 +82,10 @@ android {
     buildFeatures {
         viewBinding true
     }
+
+    lintOptions {
+        abortOnError false
```

Comment:
Why not? ;) Erroring on linking issues sounds like a good plan to enforce some common pitfalls and formatting rules...?

Reply:
It fails the builds on warnings which are pretty hard to solve and is out of the scope of the PR, you should consider removing this after an initial sweep to resolve all the lint errors

Reply:
Note that as a `# TODO`, please?